editor/editor_node.cpp

```diff
@@ -106,6 +106,7 @@
 #include "editor/export/project_zip_packer.h"
 #include "editor/fbx_importer_manager.h"
 #include "editor/filesystem_dock.h"
+#include "editor/find_in_files.h"
```

Comment:
Does this need to be included? You don't use the dialog anywhere.

Reply:
Good catch, must have missed this when changing to going through Script editor singleton, can fix this tonight